kedro-datasets/tests/snowflake/README.md

```diff
@@ -1,26 +1,8 @@
 # Snowpark connector testing
 
-Execution of automated tests for Snowpark connector requires real Snowflake instance access. Therefore tests located in this folder are **disabled** by default from pytest execution scope using [conftest.py](conftest.py).
+Execution of connection tests for Snowpark connector requires real Snowflake instance access.
```

Comment:
Is the README still relevant now the tests are part of the test suite for python < 3.12?

Reply:
No, let's delete it!